src/functionalTest/java/uk/gov/hmcts/reform/cwrdapi/client/response/RoleAdditionResponse.java

```diff
@@ -0,0 +1,13 @@
+package uk.gov.hmcts.reform.cwrdapi.client.response;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
```

Comment:
is `@Data` required? This will supply the class with `@ToString`, `@EqualsAndHashCode`, `@Getter`, `@Setter` and `@RequiredArgsConstructor` - all of these will also require mutation coverage

Reply:
These are functional tests , so do we need to care about mutation ?

Reply:
I suppose that is fine then :)